docs/usage/configuration/config_documentation.md

```diff
@@ -4033,9 +4042,9 @@ A worker can handle HTTP requests. To do so, a `worker_listeners` option
 must be declared, in the same way as the [`listeners` option](#listeners)
 in the shared config.
 
-Workers declared in [`stream_writers`](#stream_writers) will need to include a
-`replication` listener here, in order to accept internal HTTP requests from
-other workers.
+Workers declared in [`stream_writers`](#stream_writers) and [`instance_map`](#instance_map)
```

Comment:
I guess you should say 'other than `main`' somewhere here?

Reply:
You could, but this is a section about 'workers' specifically and I thought that would be TMI. Didn't want to overwhelm with information out of context.